docker/test/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM python:3.6
+FROM tonywaithaka/rappy:1
```

Comment:
There is a docker login command used in the deployment scripts that allows the docker to pull images from `gcr.io`. We can run the same command in the circle.yml file for the test job, that way we can use the gcr image instead of this one.